session-02/exercise.js

```diff
@@ -16,7 +25,21 @@ function transformArrayToUpper(listOfStrings) {
     the function should return 51
 */
 function sumOfAllAges(listOfStudentObjects) {
-  return;
+  
+  let i =listOfStudentObjects.length;
+  let totalAge =0;
+while(listOfStudentObjects.length>0)
```

Comment:
Here you want to iterate from 0 to length, instead of in-reverse (although that could work too). Using `for-of` style loops ensures you don't have index erros.